src/grammar.json

```diff
@@ -738,8 +738,124 @@
     ],
 
     "card": [
```

Comment:
I was thinking of #card# decks / #decks# with #card# (this one mostly because of verb agreement). But I don't think it's critical enough to do before merging—only suggesting it as something to keep in mind.